src/global/helpers/throttle.ts

```diff
@@ -0,0 +1,21 @@
+const throttle = <F extends (...args: any[]) => any>(
```

Comment:
should we add some test for this file? or is it enough simple?

Reply:
I don't know, I think that like the debounce it is a fairly simple function.

Reply:
Ok 👍